2-mandatory/3-stocks.js

```diff
@@ -33,8 +33,24 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+
+// 1. Find the total price of each company stocks in the last 5 days
+// 2. Divide the total price by the number of days i.e. 5
+
+
+
```

Comment:
Very good variable names 😄 
One question: Do you need to re-calculate the average each time through the loop? Maybe it's enough to just calculate the average once at the end?